tests/test_action_inputs.py

```diff
@@ -123,6 +127,10 @@ def test_get_skip_release_notes_label(mocker):
     mocker.patch("release_notes_generator.action_inputs.get_action_input", return_value="skip-release-notes")
     assert ActionInputs.get_skip_release_notes_labels() == ["skip-release-notes"]
 
+def test_get_skip_release_notes_label_not_defined(mocker):
+    mocker.patch("release_notes_generator.action_inputs.get_action_input", return_value="")
+    assert ActionInputs.get_skip_release_notes_labels() == ["skip-release-notes"]
+
```

Comment:
In this file you are mixing the one and two empty lines between the test methods. Please unite the pattern.

Reply:
Fixed in commit 3bd9b0f89377c110b90a3c003373aa1e44cc7930.